### ISSUE TITLE
ATACR Reticle incorrectly Scaled

### ISSUE THREAD

**Issue by toadie2k** [toadie2k/NIArms#146]
Apparently the MRADs on the reticle don't line up - testing that a known 9 MRAD bullet drop was consistently hitting around the 4.9-mark, so the reticle needs to be scaled@x0.54444444.

Future me, get on it.

**Comment by Blutze** [comment]
Please use that 4.9 mrad mark thing only as a rough direction and not a precise reading. I didn't bother holding the bolt animation, or aiming at a paper target, and its only 8x so I can't see that well anyway, aaaand my testing scenario doesn't completely turn off sway.

**Comment by toadie2k** [comment]
Yeah, true, but it seems _about_ consistent with what someone else said.
